secondary-clusters.md

```diff
@@ -1,13 +1,12 @@
-Secondary Clusters Guide
-===================
+# Secondary Clusters Guide
 
-Secondary clusters use a minimal Kubecost deployment to send their metrics to a central storage-bucket (aka durable storage) that is accessed by the primary cluster to provide a ___single-pane-of-glass___ view into all aggregated cluster costs globally. This aggregated cluster view is exclusive to Kubecost Enterprise.
+Secondary clusters use a minimal Kubecost deployment to send their metrics to a central storage-bucket (aka durable storage) that is accessed by the primary cluster to provide a ***single-pane-of-glass*** view into all aggregated cluster costs globally. This aggregated cluster view is exclusive to Kubecost Enterprise.
```

Comment:
Is it necessary to do both bold and italics for this term? I would go with bold as it is some unique vocab but not an interactable element.